nilearn/input_data/multi_nifti_masker.py

```diff
@@ -219,6 +219,12 @@ def fit(self, imgs=None, y=None):
         self.mask_img_.get_data()
         return self
 
+    def _check_fitted(self):
```

Comment:
As this function is common to {,Multi}NiftiMasker, you should probably put it in BaseMasker to avoid code duplication.